packages/drizzle/src/updateVersion.ts

```diff
@@ -47,7 +56,12 @@ export async function updateVersion<T extends TypeWithID>(
     select,
     tableName,
     where,
+    ignoreResult: returning === false,
   })
 
+  if (returning === false) {
+    return null
```

Comment:
If we were using `strict` this would give a typescript error.

The right way to handle this is to add overloads to all our database/types interface so that depending on the args, if you use `returning` we can make the return type null instead.

Reply:
After going back and forth on how to resolve this with an overload and not coming up with a solution, we decided to skip it for now. The returning return type does not include `null` which isn't ideal, but the greater problem with the return type is that it is actually wrong as in the case of a document not being found in the adapter currently we should have `null` but cannot add it as it would be a breaking type change.

All of this can be revisited for v4 or if there is an interim non-breaking solution we can add it still in v3 and not block this PR from being merged either way.